### DISCUSSION TITLE
Unable to type and render Unicode characters

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#163] [General]
I'm sure that the default font, `DejaVuSansMono.ttf`, supports my language.

**Comment by SpartanJ** [comment]
There are some fallback fonts for CJK languages (chinese, japanese and korean). It should work just fine, it has been confirmed by other people in the past, since it was not supported from day one, but today it's working. What's exactly not working? You cannot type it, it's not displayed? You can set a custom fallback font or also a custom monospaced font if it's needed. Which language are you trying to use?
